### PR TITLE
Fix path for channelLib

### PR DESCRIPTION
**Pull request by dieris**
Compatibility with https://github.com/BBN-Q/PyQLab/pull/115

### CODE REVIEW

File: common/util/updateNumSegments.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 args = parser.parse_args()
 
 sys.path.append(args.pyqlabpath)
-from Libraries import instrumentLib
+from QGL.ChannelLibrary import channelLib
```
Comment (caryan) [inline common/util/updateNumSegments.py]: did we want to change from `instrumentLib` to `channelLib` here?

Comment (dieris) [inline common/util/updateNumSegments.py]: actually, I don't know if this is used anywhere. This was a copy and paste error

File: common/util/updateLengthPhaseParams.py [hunk 2]
```diff
@@ -9,13 +9,15 @@
 args = parser.parse_args()
 
 sys.path.append(args.pyqlabpath)
-from Libraries import channelLib
+from QGL.ChannelLibrary import channelLib
 
 if args.qubit not in channelLib.channelDict:
 	sys.exit(1)
+	print "here"
```
Comment (blakejohnson) [inline common/util/updateLengthPhaseParams.py]: You have some debugging cruft hanging around here.

File: common/util/updateLengthPhaseParams.py [hunk 3]
```diff
@@ -9,13 +9,15 @@
...
 
 channelLib[args.qubit].pulseParams['length'] = args.length
 channelLib[args.qubit].pulseParams['phase'] = args.phase
 channelLib[args.qubit].pulseParams['amp'] = args.amplitude
 
-channelLib.write_to_file()
\ No newline at end of file
+channelLib.write_to_file()
+print "there"
```
Comment (blakejohnson) [inline common/util/updateLengthPhaseParams.py]: And here

Comment (dieris) [inline common/util/updateLengthPhaseParams.py]: oops, fixing it

**Pull request merged**